backend/src/types.ts

```diff
@@ -292,6 +292,7 @@ export type OdhApplication = {
     description: string;
     displayName: string;
     docsLink: string;
+    hidden?: boolean | null;
```

Comment:
You should add that to the frontend type too

Reply:
@lucferbux are we still syncing both `backend` and `frontend` types.

Reply:
Yes! that's right!